hack/make/build-rpm

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+set -e
+
+DEST=$1
+
+# subshell so that we can export PATH without breaking other things
+(
+	source "$(dirname "$BASH_SOURCE")/.integration-daemon-start"
+
+	# TODO consider using frozen images for the dockercore/builder-rpm tags
+		
+	rpmVersion="${VERSION//-/_}"
+	# if we have a "-dev" suffix or have change in Git, let's make this package version more complex so it works better
+	if [[ "$VERSION" == *-dev ]] || [ -n "$(git status --porcelain)" ]; then
+		gitUnix="$(git log -1 --pretty='%at')"
+		gitDate="$(date --date "@$gitUnix" +'%Y%m%d.%H%M%S')"
+		gitCommit="$(git log -1 --pretty='%h')"
+		gitVersion="git${gitDate}.0.${gitCommit}"
+		# gitVersion is now something like 'git20150128.112847.0.17e840a'
+		rpmVersion="${rpmVersion}_${gitVersion}"
```

Comment:
Euch, that's complicated to do in an automated way, and isn't at all clear that 1.3.99 is actually 1.4.  Here's my idea so far (slightly more concrete way of presenting, hopefully easier to reason about):
```
Docker 1.7.0:  version=1.7.0, release=1
Docker 1.7.0-rc1: version=1.7.0, release=0.1.rc1
Docker 1.7.0-dev nightly: version=1.7.0, release=0.0.YYYYMMDD.HHMMSS.gitHASH
```